pkg/oc/bootstrap/docker/host/host.go

```diff
@@ -136,6 +139,40 @@ func (h *HostHelper) UploadFileToContainer(src, dst string) error {
 	return err
 }
 
+//TODO: Refactor this stuff
+//TODO: write tests for both
+// DownloadFileFromContainer copies a local file to the Docker host
+func (h *HostHelper) DownloadFileFromContainer(img, src, dst string) error {
+	r, err := dockerhelper.StreamFileFromContainer(h.client, img, src)
+	if err != nil {
+		return err
+	}
+	defer r.Close()
+	data, err := ioutil.ReadAll(r)
+	if err != nil {
+		return err
+	}
+	if _, err := os.Stat(filepath.Dir(dst)); os.IsNotExist(err) {
+		err := os.Mkdir(filepath.Dir(dst), 744)
```

Comment:
0755?

Reply:
Ignore for now :) this is just a poc to confirm that stuff works. Its same permissions as current folder permissions. TODO: read current permissions and use same for our certificate. But my main concern is on approach itself (stream from running container and works only with sudo)